pkgs/top-level/perl-packages.nix

```diff
@@ -8,7 +8,7 @@
 { config
 , stdenv, lib, buildPackages, pkgs, darwin
 , fetchurl, fetchpatch, fetchFromGitHub, fetchFromGitLab
-, perl, shortenPerlShebang
+, perl, shortenPerlShebang, binlore
```

Comment:
We normally don't add packages that exist in pkgs here

Reply:
That makes sense. I just pushed a new version that uses `pkgs.binlore` instead.